src/commands/register.ts

```diff
@@ -92,25 +91,28 @@ export default class ComponentRegister extends BaseCommand {
       options.args.push({ name: 'filler' });
     }
     const parsed = await super.parse(options, argv) as Interfaces.ParserOutput<F, A>;
-    const absolute_component_path_argv = (parsed.argv || []).map(argv => path.resolve(argv));
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
-    parsed.args.component = new Set(absolute_component_path_argv);
+    parsed.args.component = parsed.argv;
 
     return parsed;
   }
 
   @RequiresDocker({ buildx: true })
   async run(): Promise<void> {
     const { args, flags } = await this.parse(ComponentRegister);
-    for (const component of args.component) {
-      const config_path = path.resolve(untildify(component));
 
+    if (!args.component || args.component.length === 0) {
+      args.component = ['./architect.yml'];
```

Comment:
We need to use `path.join('.', 'architect.yml')` here otherwise this wouldn't work on Windows properly

Reply:
`path.resolve` would also work

Reply:
Oh does that also fix paths? this may work as is then because we do call path.resolve later... I can check on my windows machine real quick to verify before Devin changes anything